README.md

```diff
@@ -2,6 +2,39 @@
 
 [![Curseforge](https://cf.way2muchnoise.eu/full_565238_downloads.svg)](https://www.curseforge.com/minecraft/modpacks/gregtech-expert-2) [![Curseforge](https://cf.way2muchnoise.eu/versions/For%20MC_565238_latest.svg)](https://www.curseforge.com/minecraft/modpacks/gregtech-expert-2) <a title="Join us on Discord!" href="https://discord.gg/xBwHpZyZdW"><img src="https://img.shields.io/discord/945647524855812176?label=GTExpert%20Discord&amp;logo=Discord&amp;style=?flat" alt="Discord"/></a> [![License: LGPL v3](https://img.shields.io/badge/License-LGPL_v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
 
+## Modpack Version Information
+### All Versions
+#### en_us
+* Major version upgrades in the following categories are not compatible.
+
+#### ja_jp
+* 下記区分の大きなバージョンアップは互換性はありません。
+
+### v1.x.x
+#### en_us
+* Stable version as of 10/28/2022 released to CurseForge. Currently, v2 is under development, and bug fixes in v1 will be included in v2.
+
+#### ja_jp
+* CurseForgeへ公開している2022/10/28現状での安定版。なお現在はv2の開発に移行しており、v1でのバグ修正はv2にて取り込まれる。
+
+### v2.x.x
+#### en_us
+* Next stable version to start releasing as a mirror on CurseForge and Github as the next version.
+* The original version included GCY:S (Gregicality Science), but the plan was changed to v3 from 2022/10/28 based on GT's development status.
```

Comment:
We have moved the original v2 which includes GCY:S (Gregicality Science) to v3, because GTCEu dev team does not have a plan to update GCY:S for next few months.